algebra/include/matrix.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Phoenix-Pilot
+ *
+ * matrix - simple library for matrix operations - header file
```

Comment:
Everyone knows that it is a header file
```suggestion
 * matrix - simple library for matrix operations 
```